public/money-out.json

```
@@ -4,34 +4,34 @@
         "createddate":"20-06-2024",
         "actiondate": "2024-06-25",
         "title": "Piano Lesson",
-        "summ": 7.00
+        "summ": "$ 7"
     },
     {
         "id": 1,
         "createddate":"20-06-2024",
         "actiondate": "2024-06-26",
         "title": "Guitar Lesson",
-        "summ": 10.00
+        "summ": "$ 10"
```

У ідеалі зберігати поряд дані з обраною валютою. Це зручніше і більш масштабовано для майбутнього. 

```suggestion
        "summ": 10,
        "currency": "USD"
```